### DISCUSSION TITLE
[2138330]Cyberpunk 2077: Phantom Liberty 赛博朋克2077：往日之影{只有DLC}

### DISCUSSION THREAD

**Discussion by yuleima** [pjy612/SteamManifestCache#605] [Share]
![2138330](https://cdn.cloudflare.steamstatic.com/steam/apps/2138330/header.jpg)

### Cyberpunk 2077: Phantom Liberty Steam 游戏ID

2138330

### 密钥列表

```hash
2138330;0b2b89913e185ab0ac452b5af584c482d5a8c6f34b7b32ae2dda130deaab64bd
2224070;13af5172058a27f8523f2caf0b4378c8dc58adf507ff3b8320ee6292ef2db899
```

### 清单附件

[2138330 mainfests](https://github.com/pjy612/SteamManifestCache/files/15012592/2138330.zip)

### 备注

之前讨论版有人发过2077本体[1091500]的库，补一下dlc的。
2138330是dlc，2224070是简中语言包

email : 17914393@qq.com
code : 995A8FD23A4D7188

**Comment by yuleima** [comment]
补一下2077本体的库吧，或许有同学会需要，放这里方便些。

app id:
`
1091500
`

keys:
```
1091501;fd4524cb30582752f15e4bf01254dbb7b890f58008101a40208cd78fb139d800
1091509;987c3b4b920a2258661e62065089d398fcc2412b300355c47a2655b4b373fafa
```
1091509为简中语言包

**Reply by barryhamsy** [reply]
thanks for this, because when i used your DLC together with someone else 2077 library, it says base game and 2077 not match.
